src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -4582,6 +4587,15 @@
     <Compile Update="C:\P\uno2\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\DatePicker\DatePickerFlyout_MinYear.xaml.cs">
       <DependentUpon>DatePickerFlyout_MinYear.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\src\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\Enability\BasicEnability.xaml.cs">
+      <DependentUpon>BasicEnability.xaml</DependentUpon>
+    </Compile>
+    <Compile Update="C:\src\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\FocusManager\FocusManager_GetFocus_Automated.xaml.cs">
+      <DependentUpon>FocusManager_GetFocus_Automated.xaml</DependentUpon>
+    </Compile>
+    <Compile Update="C:\src\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\IsEnabled\BasicEnability.xaml.cs">
+      <DependentUpon>BasicEnability.xaml</DependentUpon>
+    </Compile>
```

Comment:
Remove :)

Reply:
If I remove, we'll loose the file nesting in VS.  We need to move the <DependentUpon> at the right place in the file.  That's a lame VS is not able to do this.

Reply:
I think it is enough to just replace the absolute path with `$(MSBuildThisFileDirectory)` prefix. This makes it portable while retaining the nesting in VS